CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This changelog keeps track of all changes to the Packs SDK. We follow convention
 
 ## [Unreleased]
 
+## [1.7.0] - 2023-10-24
+
 ### Added
```

Comment:
Hmm, do we really want to include `extraOauthScopes` in here if it's not ready yet? I would move it back to Unreleased.

Reply:
Will do. I think the sdk code with this would still have been published though